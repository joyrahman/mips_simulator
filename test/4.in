ADDI R4 R0 0
ADDI R1 R0 0
SUBI R5 R1 10
BEQZ R5 13
ADDI R2 R0 0
SUBI R5 R2 10
BEQZ R5 8
ADDI R3 R0 0
SUBI R5 R3 5
BEQZ R5 3
ADDI R4 R4 1
ADDI R3 R3 1
BEQZ R0 -5
ADDI R2 R2 1
BEQZ R0 -10
ADDI R1 R1 1
BEQZ R0 -15
HLT 
