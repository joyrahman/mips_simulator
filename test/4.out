430
429
1000
570
1
3
5
65
-7
0
0
